outline.md

```diff
@@ -0,0 +1,57 @@
+## Growing an Inner Source Program
+
+45 min
+
+## Short Description
+
+"Inner source is a great idea! Go do inner source!"
+You're the dedicated champion for inner source in your company.
+What do you do now?
+How do you realistically affect the behavior of dozens or hundreds of teams to the point where robust and pervasive inner sourcing is a normal part of the way that engineering is done?
+
+This presentation shares principles, practical experience, and relatable examples on this situation.
+While not claiming all of the answers, it is insightful summary of over a year experience and work in the area.
```

Comment:
minor: probably should be: ... over a year's experience and work in the area.

Reply:
Good catch.